src/thing_objects.c

```diff
@@ -352,10 +354,11 @@ struct Objects objects_data[] = {
   {0, 0, 1, 0, 0, 901, 0x0080,    0,    0, 300, 0, 0, 2, 0,  0, ObOC_Unknown1, 0}, //133 STATUE_PLACEHOLDER6 -> SPECBOX_CUSTOM
   {0, 0, 0, 0, 1, 777, 0x0100,    0,    0, 300, 0, 0, 2, 1,  0, ObOC_Unknown1, 0}, //134 SPELLBOOK
   {0, 0, 0, 0, 1, 777, 0x0100,    0,    0, 300, 0, 0, 2, 1,  0, ObOC_Unknown1, 0}, //135
-  {0, 0, 0, 0, 0,   0, 0x0000,    0,    0,   0, 0, 0, 0, 0,  0, ObOC_Unknown0, 0},
+  {0, 0, 0, 0, 0,0x4000,0,    0,    0, 300, 0, 0, 2, 0,  0, ObOC_Unknown1, 0}, //136 TEST_THING
```

Comment:
Do all custom object needs to be added here? If so, there's not much of a point to it right?

Reply:
No. All of them are from cfg. This thing may be removed (And I actually do this in the evening)

Reply:
done